index.js

```diff
@@ -1,24 +1,24 @@
+require("dotenv").config();
 const { Client, Message, MessageEmbed, Collection } = require("discord.js");
 const fs = require("fs");
 const client = new Client({
-    messageCacheLifetime: 60,
-    fetchAllMembers: false,
-    messageCacheMaxSize: 10,
-    restTimeOffset: 0,
-    restWsBridgetimeout: 100,
-    shards: "auto",
-    allowedMentions: {
-        parse: ["roles", "users", "everyone"],
-        repliedUser: true,
-    },
-    partials: ["MESSAGE", "CHANNEL", "REACTION"],
-    intents: 32767,
+  messageCacheLifetime: 60,
+  fetchAllMembers: false,
+  messageCacheMaxSize: 10,
+  restTimeOffset: 0,
+  restWsBridgetimeout: 100,
+  shards: "auto",
+  allowedMentions: {
+    parse: ["roles", "users", "everyone"],
+    repliedUser: true,
+  },
+  partials: ["MESSAGE", "CHANNEL", "REACTION"],
+  intents: 32767,
```

Comment:
By the way, it would be better to put the intents in an Array for a better understanding

Reply:
Let's delete some Intents that are useless now and in the future
```suggestion
  intents: [
    'GUILDS',
    'GUILD_MEMBERS',
    'GUILD_BANS',
    'GUILD_EMOJIS_AND_STICKERS',
    'GUILD_VOICE_STATES',
    'GUILD_MESSAGES',
    'GUILD_MESSAGE_REACTIONS',
    'DIRECT_MESSAGES',
  ],
```